service/controller/resource/tccpn/testdata/case-0-basic-test-with-encrypter-backend-KMS-route53-enabled.golden

```diff
@@ -6,22 +6,23 @@ Outputs:
   OperatorVersion:
     Value: 7.3.0
 Resources:
+  
   ControlPlaneNodeAutoScalingGroup:
     Type: AWS::AutoScaling::AutoScalingGroup
     DependsOn: EtcdVolume
     Properties:
       VPCZoneIdentifier:
-        - subnet-id-eu-central-1b
+        - subnet-id-eu-central-1a
```

Comment:
this was caused by adding more AZs to the CP, as before there were only `eu-central-1b` and now we have `a,b,c`